### PR TITLE
add pull request template

### PR DESCRIPTION
**Pull request by 0xc0170**
There are required sections:
- description
- pull request type
- testing

I think at this stage we should start with simple template and evolve it along the way thus providing 3 sections should be sufficient as a start.

### CODE REVIEW

**Comment by evedon** [comment]
I don't think it harms to add the pull request type to indicate if the PR is a patch or new functionality.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+-->
+ 
+ 
+----------------------------------------------------------------------------------------------------------------
+### Test coverage <!-- Required -->
+ 
+<!--
+Provide all the information required, listing all the testing performed.
+-->
+[] Covered by existing automated tests
```
Comment (MarceloSalazar) [inline .github/pull_request_template.md:16]: I'd suggest we remove the line about automated tests as we don't have this at.
We require tests to run locally and to be uploaded to the PR.

Comment (0xc0170) [reply .github/pull_request_template.md:16]: OK I'll remove

Comment (0xc0170) [reply .github/pull_request_template.md]: removed in ab1b4fa

**Comment by 0xc0170** [comment]
> I don't think it harms to add the pull request type to indicate if the PR is a patch or new functionality.

Added back in 7fa4f95

**Review by MarceloSalazar — APPROVED** [review]

**Review by evedon — APPROVED** [review]

**Pull request merged**